book.rb

```diff
@@ -12,4 +12,8 @@ def initialize(author:, title:)
   def add_rental(person, date)
     Rental.new(date, self, person)
   end
+
+  def self.size
+    ObjectSpace.each_object(self).to_a
+  end
 end
```

Comment:
`README.md` file:
- [ ] Also, kindly ensure that  you have added commands on how to `clone` your repo, `access` the cloned directory, `run your project locally`... in your README file to help non technical users on how to proceed. I suggest you refer this **[example](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)** to have an idea

- [ ] I think it would be better if you fix the broken `Live Demo`  in your README file to make it more professional as it shows a **404** error page once clicked. As you will not deploy your app online, I can suggest you leave there an information such as: `N/A`, `Not Available`,...